settings.gradle

```diff
@@ -3,6 +3,7 @@ include ':base'
 include ':messagingpush'
 include ':common-test'
 include ':messaginginapp'
+include ':messaginginapp-compose'
```

Comment:
Not related to this file, but can we please include `messaginginapp-compose` module in SDK binary size task as well?

Reply:
good call out